### PR TITLE
Add FreedomFi maker

### PR DESCRIPTION
**Pull request by IgoninVV**

### CODE REVIEW

**Comment by cokes518** [comment]
looks good to me. 

Before we merge we'll need to do 2 things:

- [ ] confirm HW audit passed
- [ ] create a maker account so we can replace the `id` with the correct maker account `id`.

File: src/makers/freedomfi/hotspots.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows FreedomFi to identify issues with your Hotspot in a secure way.</white></b>\n\nFreedomFi will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@freedomfi.com</b></purple> from the email used to purchase the Hotspot.',
+      power: [
+        "Attach the antenna's and plug in the provided power adapter.",
+        'The FreedomFi PWR LED will light up once it’s powered on.',
+      ],
+      externalOnboard:
+        'Please onboard your FreedomFi hotspot by tapping the link below:',
```
Comment (cokes518) [inline src/makers/freedomfi/hotspots.ts:18]:
```suggestion
        'Please onboard your FreedomFi Hotspot by tapping the link below:',
```

**Review by cokes518 — COMMENTED** [review]
capitalization change requested, but lgtm

**Review by cokes518 — APPROVED** [review]
ready for merge @tyler-whitman @matthewcarlreetz

**Review by tyler-whitman — APPROVED** [review]
looks good!

**Pull request merged**